container_ci_suite/helm.py

```
@@ -188,6 +190,7 @@ def is_pvc_in_values_yaml(self) -> bool:
             True if the PVC is in the values.yaml file, False otherwise
         """
         chart_dict = utils.get_yaml_data(self.full_package_dir / "values.yaml")
+        logger.debug
```

Ruff (0.15.15)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)